tests/engines/test_torch.py

```diff
@@ -100,7 +100,7 @@ def test_embeddings() -> None:
         "orig_param_values", {}
     )  # TODO: remove this when embedding system is updated
 
-    assert len(list(low_level_params.keys())) == 9
+    assert len(list(low_level_params.keys())) == 9 + len(inputs)
```

Comment:
Are you sure this is the right fix?

The line above has a comment to be removed "when embedding system is updated". Could it be that removing that will make the `assert` pass without the extra manual addition?

Reply:
good catch, I'll update the test

Reply:
It turns out that I already removed the `orig_param_values` key, thus the test is actually correct.

Reply:
I just removed the unnecessary code from these tests.